### ISSUE TITLE
Error: Class 'ProcessWire\GraphQL\Field\WireArray\WireArrayCountField' not found

### ISSUE THREAD

**Issue by PatricB** [dadish/ProcessGraphQL#18]
The Class **'PagesCountField'** in [ProcessGraphQL/src/Field/Pages/PagesCountField.php](https://github.com/dadish/ProcessGraphQL/blob/master/src/Field/Pages/PagesCountField.php) should extend **'ProcessWire\GraphQL\Field\WireArray\WireArrayCountField'**, but I could not find a Class wit this name in the plugin.
Seems like something is missing in the Plugin or I am missing something.

**Comment by dadish** [comment]
You are right. It does miss the required class. I forgot to implement tests for it. It comes with the `pages` field, which is experimental. But I'll fix it later this week and update you here.

**Comment by dadish** [comment]
@PatricB I made some updates regarding this bug. Please check the [latest release](https://github.com/dadish/ProcessGraphQL/releases/latest) and let me know if the issue is gone.
